extensions/ui/org.eclipse.smarthome.ui.paper/web-src/js/services.repositories.js

```diff
@@ -148,7 +148,11 @@ angular.module('PaperUI.services.repositories', []).factory('bindingRepository',
             repository.add(discoveryResult);
         }
         if (topic.indexOf("removed") > -1 && index != -1) {
-            repository.remove(discoveryResult, index);
+            repository.remove(undefined, index);
+        }
+        if (topic.indexOf("updated") > -1 && index != -1) {
```

Comment:
I know, it's been like that previously with the `removed` as well, but still - what happens if I (or rather: a binding) name(s) a Thing e.g. `mysterious_thing_updated`? When this now gets updated, the topic would be `smarthome/inbox/mysterious_thing_updated/removed`. Enjoy debugging 😬 
What I actually want to say: The pure string comparisons here are a little too simple.

Reply:
agreed, the current string matching is not enough. I will update the regex to better match the actual topic.